gnomad_qc/v4/resources/sample_qc.py

```diff
@@ -5,9 +5,16 @@
     VersionedMatrixTableResource,
     VersionedTableResource,
 )
-from gnomad.sample_qc.relatedness import get_relationship_expr
+from gnomad.sample_qc.relatedness import (
+    UNRELATED,
+    SECOND_DEGREE_RELATIVES,
+    PARENT_CHILD,
+    SIBLINGS,
+    DUPLICATE_OR_TWINS,
+    AMBIGUOUS_RELATIONSHIP,
+)
```

Comment:
Just a note that we order the imports alphabetically

Reply:
Thanks, I'm surprised `black` doesn't do that automatically. Fixed now.